src/analytics.ts

```diff
@@ -60,4 +60,26 @@ export const Event = {
   HelpFeedback: 'charts_help_feedback',
 
   IncompleteDataUpgrade: 'charts_incomplete_data_upgrade',
+
+  AlertsUsage: 'alerts_usage',
+  ClickShareAlert: 'alerts_share',
+  ClickCopyAlertLink: 'alerts_copy_link',
+  ClickCopyAlert: 'alerts_copy',
+  OpenAlert: 'alerts_open',
+  ClickCreateAlert: 'alerts_create',
+  ChangeAlertAbility: 'alerts_change_ability',
+  ChangeAlertVisibility: 'alerts_change_visibility',
+  ClickEditAlert: 'alerts_edit',
+  SetAlertType: 'alerts_set_type',
+  SetAlertAsset: 'alerts_set_asset',
+  SetAlertWatchlist: 'alerts_set_watchlist',
+  SetAlertScreener: 'alerts_set_screener',
+  SetAlertAddress: 'alerts_set_address',
+  SetAlertWalletAsset: 'alerts_set_address_asset',
+  SetAlertSocialTrend: 'alerts_set_social',
+  SetAlertMetric: 'alerts_set_metric',
+  SetAlertCondition: 'alerts_set_condition',
+  SetAlertChannel: 'alerts_set_channel',
+  CreateAlert: 'alerts_save_new',
+  EditAlert: 'alerts_save_old',
```

Comment:
`san-studio` has events related to  the charts module. Better keep this separation. Create such variable in the sanbase repository